azkaban-webserver/src/main/resources/azkaban/webapp/servlet/velocity/executingflowpage.vm

```diff
@@ -112,6 +112,9 @@
         <li class="nav-button pull-right"><button type="button" id="cancelbtn" class="btn btn-danger btn-sm">Kill</button></li>
         <li class="nav-button pull-right"><button type="button" id="retrybtn" class="btn btn-success btn-sm">Retry Failed</button></li>
         <li class="nav-button pull-right"><button type="button" id="executebtn" class="btn btn-success btn-sm">Prepare Execution</button></li>
+        #if ($executionExternalLinkURL)
```

Comment:
Nope, Dr. Elephant will be the first button. The 'pull-right' bootstrap class will cause the first 'li' element to take the right most spot. As this is mentioned as the last line this will appear as the first button.